build/manifest/main.go

```diff
@@ -114,7 +114,7 @@ func applyManifest(manifest *model.Manifest) error {
 
 		// write generated code to file by using Go file template.
 		if err := ioutil.WriteFile(
-			"server/plugin/manifest.go",
+			"server/manifest.go",
```

Comment:
Is that change intended? `server/manifest.go` doesn't seem to be checked in.

Reply:
Good catching 👍  Intended, but this doesn't seem to make sense. I revert it.